book/from_flat_php_to_symfony2.rst

```diff
@@ -557,10 +581,15 @@ them for you. Here's the same sample application, now built in Symfony::
         }
     }
 
+First we have a "controller class" which is a convenient way to group several
+"controllers" together. So methods inside a controller class are controllers
+also called *actions*. They hold code which creates and returns the appropriate
+``Response`` object.
```

Comment:
I added this based on my personal experience... When i was introduced to Symfony and start reading the documentation i had trouble understanding this chapter and the controller concept - diff between flat PHP and Symfony controllers - because of this missing information that didn't appeared until in some later chapter.. World controller is used thought the chapter and it does not always refer to the same thing. So i think that to a newcomer it's quite unclear if it doesn't have any background from before and essential to understand what it's actually meant.
Do you think i should still through this out?